### ISSUE TITLE
Update `morpho-data-structures` submodule once open-source

### ISSUE THREAD

**Issue by Rubilmax** [morpho-org/morpho-aavev3-optimizer#734]
Once `morpho-data-structures-private` is merged into [morpho-dao/morpho-data-structures](https://github.com/morpho-dao/morpho-data-structures):
- [x] [Remove `morpho-data-structures-private` submodule](https://gist.github.com/myusuf3/7f645819ded92bda6677)
- [x] `forge install morpho-dao/morpho-data-structures`
